### PR TITLE
os-02: Fix for SUSE environments

### PR DESCRIPTION
**Pull request by mcgege**
Missing shadow_group + test for SUSE

### CODE REVIEW

File: controls/os_spec.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@
 login_defs_passmaxdays = attribute('login_defs_passmaxdays', default: '60', description: 'Default password maxdays to set in login.defs')
 login_defs_passmindays = attribute('login_defs_passmindays', default: '7', description: 'Default password mindays to set in login.defs')
 login_defs_passwarnage = attribute('login_defs_passwarnage', default: '7', description: 'Default password warnage (days) to set in login.defs')
-if os.redhat?
+if os.redhat? || os.suse?
   shadow_group = 'root'
 elsif os.debian?
   shadow_group = 'shadow'
```
Comment (artem-sidorenko) [inline controls/os_spec.rb]: @chris-rock @atomic111 we do not have here a sane default. It means in the situation prior to this change on a suse system shadow_group would be `nil`. How should we handle this? Options from my POV:

1. introduction of else and raise, something like:
```ruby
shadow_group = if os.redhat?
                              'root'
                            elsif
                              'shadow'
                            else
                              raise 'Unsupported platform'
                            end
```

2. defined sane default, like

```ruby
shadow_group = 'root'
if os_debian?
  shadow_group = 'shadow'
end
```

Ideas, opinions?

**Comment by mcgege** [comment]
Had to rework my patch as the default group for /etc/shadow is "shadow" and group readable on a fresh SUSE system

File: controls/os_spec.rb (hunk 1 above)
Comment (chris-rock) [inline controls/os_spec.rb]: The correct way is 1., but InSpec does not support exceptions yet. This is a high-prio ticket for one of the next InSpec releases. Should we go with 2. for now?

Comment (mcgege) [inline controls/os_spec.rb]: @chris-rock As I had to rework my patch I already implemented the second option ...

**Comment by artem-sidorenko** [comment]
@mcgege and my bad, I totally forgotten to ask you to sign-off your commits :) (and we do not have a bot yet), could you please [sign-off](https://stackoverflow.com/questions/1962094/what-is-the-sign-off-feature-in-git-for) your future commits?

**Comment by mcgege** [comment]
@artem-sidorenko done

**Pull request merged**